### PR TITLE
Remove CSR in favor of Service Account

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: scripts/src/primazactl/kubeidentity/identity.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import polling2
```
Comment (mmulholla) [inline scripts/src/primazactl/kubeidentity/identity.py]: This file appears to be redundant.

File: scripts/src/primazactl/identity/kubeidentity.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        :type kubeconfig: KubeConfigWrapper
+        :param kubeconfig: Kubeconfig to use to connect to the cluster
+        :type identity: str
+        :param identity: The name of the identity
+        :type namespace: str
+        :param namespace: Namespace where to create the identity
+    """
+    corev1 = client.CoreV1Api(api_client)
+
+    sa = client.V1ServiceAccount(
```
Comment (mmulholla) [inline scripts/src/primazactl/identity/kubeidentity.py:92]: For this I would like to see a re-useable ServiceAccount object with create, read and delete functions. This then becomes re-usable in other cases when we create a service account object. I can change to this as part of create namespace work if you prefer.

**Review by mmulholla — COMMENTED** [review]
Functionally looks good but would like a slightly different structure to create reusable service account and secret objects, but I can do that refactor as part if the create namespace work.

File: scripts/src/primazactl/kubeidentity/identity.py (hunk 1 above)
Comment (filariow) [reply scripts/src/primazactl/kubeidentity/identity.py]: great catch! that's a result of some tests ans was not meant to be pushed. Let me remove this.

File: scripts/src/primazactl/identity/kubeidentity.py (hunk 2 above)
Comment (filariow) [reply scripts/src/primazactl/identity/kubeidentity.py:92]: Ok, let merge as it is and discuss the refactor you're proposing in a new PR

**Review by mmulholla — APPROVED** [review]

**Pull request merged**